src/System.Private.CoreLib/src/System/Environment.Uap.cs

```diff
@@ -38,15 +38,6 @@ public unsafe static String GetEnvironmentVariable(String variable)
             return null;
         }
 
-        public static string MachineName
```

Comment:
It is ok to remove this without breaking anything because of this method is not in the UAP surface.